lite/examples/style_transfer/android/app/download_model.gradle

```diff
@@ -4,24 +4,38 @@ task downloadPredictModelFile(type: Download) {
     overwrite false
 }
 
-task downloadTransformModelDynamicFile(type: Download) {
-    src 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/arbitrary_style_transfer/style_transfer_quantized_dynamic.tflite'
-    dest project.ext.ASSET_DIR + '/style_transfer_quantized_dynamic.tflite'
-    overwrite false
-}
+// TODO: Add support for dynamic size model
+//task downloadTransformModelDynamicFile(type: Download) {
+//    src 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/arbitrary_style_transfer/style_transfer_quantized_dynamic.tflite'
+//    dest project.ext.ASSET_DIR + '/style_transfer_quantized_dynamic.tflite'
+//    overwrite false
+//}
 
 task downloadTransformModelFile(type: Download) {
     src 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/arbitrary_style_transfer/style_transfer_quantized_384.tflite'
     dest project.ext.ASSET_DIR + '/style_transfer_quantized_384.tflite'
     overwrite false
 }
 
+task downloadPredictGPUModelFile(type: Download) {
```

Comment:
How about naming it as `downloadPredictFloat16ModelFile`?
Technically it could be accelerated by other hardwares, too.

Reply:
Done.